### ISSUE TITLE
Continuous integration: Fleet vs pipelines / repos

### ISSUE THREAD

**Issue by mrmedicine** [rancher/rancher#31840]
In Rancher 2.5.6 (in the cluster manager) I have the great option to go to Pipelines and add a git repo, even select an example repo.

E.g. "https://github.com/rancher/pipeline-example-php.git". In turn this will automatically build the source repo (using Jenkins) and this is exactly what I look forward to use in Rancher. However its seems this is soon to be deprecated:

> Cluster Manager - Rancher Pipelines | Git-based deployment pipelines is now recommend to be handled with   Rancher Continuous Delivery powered by Fleet available in Cluster  Explorer.
> -- | --

I am at a loss how to replicate the previous behavior i.e checking in to source code and building / releasing the image with Rancher Fleet. _Can someone point me in the right direction?_ I have a few ideas on how to accomplish this but I would like to follow recommendations. The scaling options (multicluster) of Fleet are awesome btw.

**Comment by sangnguyen7** [comment]
I have the same question. How do we achieve the same capability of the old pipeline in the new fleet?

**Comment by jvdbrink-netpulse** [comment]
> 
> 
> I have the same question. How do we achieve the same capability of the old pipeline in the new fleet?

@sangnguyen7 In the meantime I have manually replicated the behavior by making use of gitlab / a gitlab runner to build the images with kaniko and push them to a registry / repo. Once the image is build I've instructed FluxCD to monitor the docker registry for changes through what is called a ImageUpdateAutomation. In the end I'm happy with what I learned in gitops and how it turned out although it was time consuming. 

I've also answered my own question regarding this topic on stackoverflow:
https://stackoverflow.com/questions/66859165/fleet-continuous-delivery-build

TLDR: My conclusion is that fleet is a great tool (especially if you manage many clusters) but does not provide a full CI/CD solution as Rancher pipelines did, in fact it does not even come close. –
